getting-started/setup-multi-node-auth.md

```diff
@@ -66,33 +158,42 @@ Then apply the correct permissions to the file:
 chmod 0600 passfile
 ```
 
-### 3. Reload server configuration
+#### 3. Reload server configuration
 Reload the server configuration on each node for the changes to take effect:
 
 ```bash
 pg_ctl reload
 ```
 
-##### 4. Add users to all nodes
-While connected to the access node:
+#### 4. Attach the data nodes to the access node
 
-```sql
-CREATE ROLE alice WITH LOGIN PASSWORD 'foobar';
-```
+Once the nodes are properly configured, you can continue following the [multi-node setup][attach_multi_node_nodes].
 
-Note: The `postgres` user should already exist, so shouldn't need to be added to each node.
+#### 5. Setting up additional roles [](multi-node-auth-password-roles)
 
-Then, to create the same user on all data nodes, use the `distributed_exec` function 
-([API docs][distributed_exec]) from the access node:
+This is similar to adding roles with trust authentication [above](multi-node-auth-trust-roles),
+but there are a few extra steps needed.
```

Comment:
I don't think it is needed to refer to the trust authentication, since all commands are given here.

Reply:
Good point, removed